modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/components/sidebar/MultiPanelSidebarFormsProxy.es.js

```diff
@@ -22,32 +22,39 @@ import DragLayer from '../../drag-and-drop/DragLayer.es';
 import MultiPanelSidebar from './MultiPanelSidebar.es';
 
 const MultiPanelSidebarFormsProxy = React.forwardRef(
-	({
-		activePage,
-		dataProviderInstanceParameterSettingsURL,
-		dataProviderInstancesURL,
-		defaultLanguageId,
-		editingLanguageId,
-		fieldTypes,
-		focusedField,
-		functionsMetadata,
-		functionsURL,
-		instance,
-		onChange,
-		pages,
-		panels,
-		rules,
-		sidebarPanels,
-		sidebarVariant,
-		spritemap,
-	}) => {
+	(
+		{
+			activePage,
+			dataProviderInstanceParameterSettingsURL,
+			dataProviderInstancesURL,
+			defaultLanguageId,
+			editingLanguageId,
+			fieldTypes,
+			focusedField,
+			functionsMetadata,
+			functionsURL,
+			instance,
+			onChange,
+			pages,
+			panels,
+			rules,
+			sidebarPanels,
+			sidebarVariant,
+			spritemap,
+		},
+		forwardRef
+	) => {
 		const [{currentPanelId, open}, setStatus] = useState({
 			currentPanelId: 'fields',
 			open: true,
 		});
 
 		return (
-			<DndProvider backend={HTML5Backend} context={window}>
+			<DndProvider
+				backend={HTML5Backend}
+				context={window}
+				ref={forwardRef}
```

Comment:
We shouldn't never pass a `ref` to a provider component. Why not directly on MultiPanelSidebar?

Reply:
Sure thing! I'll do that.